src/lib/snack-bar/simple-snack-bar.ts

```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {BaseSnackBarContent} from './base-snack-bar';
+
+
+@Component({
```

Comment:
Add class description explaining what this component is used for, also mentioning that it is internal only.